code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -531,7 +531,7 @@
 		to_chat(usr, "<span class='warning'>Error: Embedded Syndicate credentials contain an abnormal signature. Aborting.</span>")
 		return
 
-	var/obj/item/I = ..()
+	var/obj/item/I = new item(loc)
```

Comment:
im a terrible reviewer, it should be like this so theyre still charged for the thing. i.e. revert it back to what it was, my bad.
```suggestion
	var/obj/item/I = ..()
```